package.json

```diff
@@ -14,11 +14,14 @@
   },
   "private": true,
   "dependencies": {
+    "@angular-material-extensions/password-strength": "^8.0.0",
     "@angular/animations": "~10.1.5",
     "@angular/cdk": "^10.2.7",
     "@angular/common": "~10.1.5",
     "@angular/compiler": "~10.1.5",
     "@angular/core": "~10.1.5",
+    "@angular/fire": "^6.1.4",
+    "@angular/flex-layout": "^11.0.0-beta.33",
```

Comment:
Any reason for adding a beta dependency?

Reply:
They have all their versions with a beta suffix. Ngx-auth-firebaseui has this dependency.

Reply:
As per the [docs](https://github.com/AnthonyNahas/ngx-auth-firebaseui/#peer-dependencies---please-make-sure-that-peerdependencies-are-installed-if-you-are-not-using-the-schematics) we should be using `ngx-auth-firebaseui` v4

> v1.x should be used with angular v7 projects v3.x should be used with angular v8 projects **v4.x should be used with angular v9/10 projects** v5.x should be used with angular v11 projects

But we have v5 installed and also its peerDependencies. Should we also downgrade `ngx-auth-firebaseui`?

Reply:
Good catch.  Trying to get it downgraded but node modules is breaking...lemme see

Reply:
The peer dependencies didn't work properly with the app. Some of the dependencies uses `commonJS` which will always result in a warning from Angular 10 and above. Angular docs recommends [ES Modules](https://angular.io/guide/build#configuring-commonjs-dependencies). However, this will still let us compile the code albeit a warning. But during testing I found the authentication breaking. Tried some other combinations of versions - didn't work. I have a feeling those versions might have worked perfectly on Angular 9.

Reply:
We can open a task if we want to investigate further.